website/docs/docs/collaborate/explore-projects.md

```diff
@@ -51,75 +47,107 @@ By default, dbt Explorer shows the project's [applied state](/docs/dbt-cloud-api
+</expandable>
 
-To browse using a different view, you can choose one of these options from the **View by** dropdown:
+<expandable alt_header="Filters side panel">
 
-- **Resources** (default) &mdash; All resources in the project organized by type.
-- **Packages** &mdash; All resources in the project organized by the dbt package in which they are defined.
-- **File Tree** &mdash; All resources in the project organized by the file in which they are defined. This mirrors the file tree in your dbt project repository.
-- **Database** &mdash; All resources in the project organized by the database and schema in which they are built. This mirrors your data platform's structure that represents the [applied state](/docs/dbt-cloud-apis/project-state) of your project. 
+The **Filters** side panel becomes available after you perform a keyword search. Use this panel to further refine the results from your keyword search. By default, Explorer searches across all resources in the project. You can filter on:
 
-<LoomVideo id='3bd3b16b22ab49ea809c9a863be06d05' />
+- [Resource type](/docs/build/projects) (like models, sources, and so on)
+- [Model access](/docs/collaborate/govern/model-access) (like public, private)
+- [Model layer](/best-practices/how-we-structure/1-guide-overview) (like marts, staging)
+- [Model materialization](/docs/build/materializations) (like view, table)
+- [Tags](/reference/resource-configs/tags) (supports multi-select)
+
+Under the the **Models** option, you can filter on model properties (access or materialization type). Also available are **Advanced** options, where you can limit the search results to column name, model code, and more.  
+
+</expandable>
+
+### Example of keyword search
+Example of results from searching on the keyword`item` and applying the filters models, description, and code:
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/example-keyword-search.png" width="100%" title="Example of keyword search" />
+
+
+## Browse with the sidebar
+
+From the sidebar, you can browse your project's resources, its file tree, and the database. 
+
+- **Resources** tab &mdash; All resources in the project organized by type. Select any resource type in the list and all those resources in the project will display as a table in the main section of the page. For a description on the different resource types (like models, metrics, and so on), refer to [About dbt projects](/docs/build/projects).
+- **File Tree** tab &mdash; All resources in the project organized by the file in which they are defined. This mirrors the file tree in your dbt project repository.
+- **Database** tab &mdash; All resources in the project organized by the database and schema in which they are built. This mirrors your data platform's structure that represents the [applied state](/docs/dbt-cloud-apis/project-state) of your project. 
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/example-search-sidebar.png" title="Example of tabs in sidebar" />
```

Comment:
The other screenshots are fine, but this one is off-center in its text. Maybe a Docusaurus thing?

<img width="462" alt="Screenshot 2024-03-28 at 5 06 49 PM" src="https://github.com/dbt-labs/docs.getdbt.com/assets/60105315/da2ab858-f3f0-43cb-bf3d-f692277ad428">

Reply:
filed asana ticket